web/src/beta/lib/core/Map/types/index.ts

```diff
@@ -266,6 +266,10 @@ export type TerrainProperty = {
   terrainCesiumIonUrl?: string;
   terrainUrl?: string;
   terrainNormal?: boolean;
+  heatmap?: boolean;
```

Comment:
```suggestion
  heatmap?: boolean;
  heatmapEllipsoid?: "japan-sea-level";
```

Reply:
You need to pass `heatmapEllipsoid` from [here](https://github.com/reearth/reearth/blob/9b9d4bcf792bc7bc374ff32fb66b7ce2c2671219/web/src/beta/lib/core/engines/Cesium/hooks.ts#L467) like below.
```
useOverrideGlobeShader({
  // ...
  heatmapEllipsoid: property?.terrain?.heatmapEllipsoid,
});
```